gp-templates/glossary-view.php

```diff
@@ -68,7 +68,7 @@
 
 				<form action="<?php echo esc_url( gp_url_join( $url, '-new' ) ); ?>" method="post">
 					<dl>
-						<dt><label for="new_glossary_entry_term"><?php _ex( 'Original term:', 'glossary entry', 'glotpress' ); ?></label><dt>
+						<dt><label for="new_glossary_entry_term"><?php esc_html( 'Original term:', 'glossary entry', 'glotpress' ); ?></label></dt>
```

Comment:
You replaced a translation function with an escaping function, if there was such a think you would have used `esc_html_ex()`, but since that doesn't exist you'll just have to wrape `_ex()` with `esc_html()`.